cyder/core/cyuser/backends.py

```diff
@@ -58,37 +58,52 @@ def _has_perm(user, ctnr, action, obj=None, obj_class=None):
     if user.is_superuser:
         return True
 
+    ctnr_level = -1
+    assert LEVEL_ADMIN > LEVEL_USER > LEVEL_GUEST > ctnr_level
+
+    if obj:
+        try:
+            ctnrs = obj.get_ctnrs()
+        except TypeError:
+            ctnrs = None
```

Comment:
Lets say our obj is Nameserver, we get a TypeError, ctnr_level stays at -1, leaving ctnr_guest to = false. Is this intended in the case of a typeerror?